src/app/dashboard/rankingCard/RankingCard.tsx

```diff
@@ -0,0 +1,29 @@
+import { Box, LinearProgress, Typography } from '@mui/material';
+import EventsIcon from '@mui/icons-material/EmojiEvents';
+
+import { styles } from './RankingCard.styles';
+import { RankingCardProps } from './RankingCard.types';
+import { useLocale } from 'hooks';
+import { Card } from 'ui';
+
+export const RankingCard = ({ name, score, rank }: RankingCardProps) => {
+  const { formatMessage } = useLocale();
+
+  const progressPercent = Math.round((score / 2222) * 100);
```

Comment:
(tu i poniżej) Te 2222 to taki magic number, i to pojawiający się kilka razy (więc znowu - DRY) ;) To przyjdzie z API, ale na ten moment można to wyciągnąć jako zwykły const przed definicją komponentu, np jako `const GOAL = 2222;`